modules/bitgo/src/v2/internal/tssUtils.ts

```diff
@@ -314,7 +315,7 @@ export class TssUtils extends MpcUtils {
       backupKeyShare,
       bitgoKeychain,
       params.passphrase,
-      params.originalPasscodeEncryptionCode
+      params.originalPasscodeEncryptionCode!
```

Comment:
we should update `originalPasscodeEncryptionCode?: string;` in input to be non-optional

Reply:
oh tru. i did this cus its not needed for backup/bitgo, but since all of them are created under this fn its fine.